version/version.go

```diff
@@ -1,4 +1,4 @@
 package version
 
 // Version is the version of the build.
-const Version = "1.0.9-dev"
+const Version = "1.0.10-dev"
```

Comment:
Is this version bump intentional?  I don't see it discussed in the PR comments.

Reply:
yup, it's intended sorry, it came up just now

Reply:
> yup, it's intended sorry, it came up just now

Shouldn't [the master branch be using a version number > 1.9.x][1] if it uses a version number at all?

[1]: https://github.com/kubernetes-incubator/cri-o/blame/28997fe4cde12ee5808044f55f2014df97f31a6e/README.md#L16

Reply:
This PR is targeting the `release-1.7` branch.

Reply:
> This PR is targeting the `release-1.7` branch.

Ah, I hadn't realized it was a (partial?) [backport][1] of #1154.

[1]: https://github.com/kubernetes-incubator/cri-o/pull/1154/files#diff-702431da3a2e35392cf253955d15f774R433